pkg/cmd/miscellaneous.go

```diff
@@ -274,18 +275,14 @@ func getEmail(githubURL string) string {
 	if githubURL == "" {
 		return "null"
 	}
-	res, err := ExecCmdReturnOutput("bash", "-c", "curl -ks "+githubURL+"/api/v3/users/"+os.Getenv("USER"))
+	resp, err := http.Get(githubURL + "/api/v3/users/" + os.Getenv("USER"))
```

Comment:
```go
resp, err := http.Get(githubURL + "/api/v3/users/" + os.Getenv("USER"))
```
Please do url encoding, e.g. checkout https://www.urlencoder.io/golang/ which explains it nicely

Reply:
Hi @petersutter , thanks for the comments! i changed these two functions according to link you provided.
thanks
-Neo